index.html

```diff
@@ -96,10 +100,25 @@ <h1>The OpenTF Manifesto</h1>
         </ul>
 
         <p class="font-bold">
-          We ask HashiCorp to do right by the community and donate Terraform to
-          a reputable foundation where, together with the community and
-          resources that we pledge, we continue to build and maintain this
-          amazing ecosystem.
+          Our request of HashiCorp: donate Terraform to this foundation.
+        </p> 
+        
+        <p>
+          We ask HashiCorp to do the right thing by the community: donate Terraform to this foundation,
```

Comment:
L 103/107 restatement back-to-back briefly confused me and caused me to reread the section

Reply:
Slightly rephrased: https://github.com/opentffoundation/manifesto/pull/20/commits/124b16c35d9576871b42e3336bf967e8f57bd8fa. LMK if clearer.